conf/app.routes

```diff
@@ -106,6 +106,7 @@ POST        /new-liability                                      @uk.gov.hmrc.tar
 GET         /cases/:reference/keywords/:keyword                 @uk.gov.hmrc.tariffclassificationfrontend.controllers.CaseController.removeKeyword(reference: String, keyword: String)
 
 GET         /pdf/ruling/:reference                              @uk.gov.hmrc.tariffclassificationfrontend.controllers.PdfDownloadController.rulingPdf(reference: String)
+GET         /pdf/decision/:reference                            @uk.gov.hmrc.tariffclassificationfrontend.controllers.PdfDownloadController.liabilityPdf(reference: String)
```

Comment:
Could we try have just one endpoint with a case type switch? (Combine ruling and decision)

Reply:
I should think we can manage that